Framework/Core/src/CommonServices.cxx

```diff
@@ -527,22 +527,34 @@ auto sendRelayerMetrics(ServiceRegistry& registry, DataProcessingStats& stats) -
   stats.channelBytesIn.resize(device->fChannels.size());
   stats.channelBytesOut.resize(device->fChannels.size());
   size_t ci = 0;
+  auto lastTotalBytesIn = stats.totalBytesIn.load();
+  auto lastTotalBytesOut = stats.totalBytesOut.load();
+  stats.totalBytesIn = 0;
+  stats.totalBytesOut = 0;
+
   for (auto& channel : device->fChannels) {
     auto newBytesOut = channel.second[0].GetBytesTx();
     auto newBytesIn = channel.second[0].GetBytesRx();
     monitoring.send(Metric{(float)(newBytesOut - stats.channelBytesOut[ci]) / 1000000.f / (timeSinceLastLongUpdate / 1000.f),
-                           fmt::format("channel_{}_rate_in_mb_s", channel.first),
+                           fmt::format("channel_{}_rate_out_mb_s", channel.first),
                            o2::monitoring::Verbosity::Debug}
                       .addTag(Key::Subsystem, Value::DPL));
     monitoring.send(Metric{(float)(newBytesIn - stats.channelBytesIn[ci]) / 1000000.f / (timeSinceLastLongUpdate / 1000.f),
-                           fmt::format("channel_{}_rate_out_mb_s", channel.first),
+                           fmt::format("channel_{}_rate_in_mb_s", channel.first),
                            o2::monitoring::Verbosity::Debug}
                       .addTag(Key::Subsystem, Value::DPL));
+    stats.totalBytesIn += newBytesIn;
+    stats.totalBytesOut += newBytesOut;
     stats.channelBytesOut[ci] = newBytesOut;
     stats.channelBytesIn[ci] = newBytesIn;
     ci++;
   }
 
+  monitoring.send(Metric{(float)(stats.totalBytesOut - lastTotalBytesOut) / 1000000.f / (timeSinceLastLongUpdate / 1000.f), "total_rate_out_mb_s"}
```

Comment:
If I understand correctly the latency for aggregating `timeSinceLastLongUpdate` is minimum 30 seconds. I'd actually prefer a shorter granularity for the total in/out metric, perhaps 10 seconds?
I don't care so much for the per-channel metrics.

Reply:
ok, I will move it to be on the 5 seconds interval.